test/MathOptInterface/MOI_wrapper.jl

```diff
@@ -39,6 +39,23 @@ end
     @test MOIU.supports_default_copy_to(OPTIMIZER, true)
 end
 
+@testset "branching_priority" begin
+    coluna = JuMP.optimizer_with_attributes(
+        Coluna.Optimizer,
+        "params" => CL.Params(solver=ClA.SolveIpForm()),
+        "default_optimizer" => GLPK.Optimizer
+    )
+    model = BlockModel(coluna, direct_model=true)
+    @variable(model, x)
+    @constraint(model, x <= 1)
+    @objective(model, Max, x)
+    optimize!(model)
+    
+    for (moi_index, varid) in backend(model).varids
+        @test MOI.get(backend(model), Coluna.VarBranchingPriority(), moi_index) == 1
+    end
```

Comment:
I think you can just do :

```suggestion
    @test MOI.get(model, Coluna.VarBranchingPriority(), x) == 1
```

Reply:
No, it doesn't. Should `x` be a `MOI.VariableIndex`? It appears to be a `JuMP.VariableRef`.